src/parser.ts

```diff
@@ -7,6 +7,14 @@ export type ParseParams = {
   seen: { schema: any; objects: any[] }[];
 };
 
+const ZodErrorHandler = (messageOrObject: string | object): void => {
```

Comment:
I am not familiar with the `zod` codestyle rules, but it is better to name functions int he camelCase style

```
zodErrorHandler
```

Reply:
I have following styles in the repo looks like it is ok